tests/integration/cli/test_cli_entrypoint.py

```
@@ -0,0 +1,28 @@
+    test_args = [
+        "main.py",
+        "--env-config",
+        env_path,
+        "--model-config",
+        model_path,
+        "--trainer-config",
+        trainer_path,
+    ]
+    with (
+        patch.object(sys, "argv", test_args),
+        patch("src.main.Trainer") as mock_trainer_class,
```

Ruff (0.11.9)

Local variable `mock_trainer_class` is assigned to but never used

Remove assignment to unused variable `mock_trainer_class`

(F841)